### PR TITLE
Update configuration with partial code refactoring.

### PR DESCRIPTION
**Pull request by kegor**
Replicated changes from https://github.com/groupdocs-total/GroupDocs.Total-for-NET-MVC/pull/94 with partial code style refactoring using StyleCop.

### CODE REVIEW

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 9
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5681088&bid=18596657)

File: src/Products/Annotation/Controllers/AnnotationApiController.cs [hunk 1]
```diff
@@ -12,168 +21,180 @@
...
     {
-        private static Common.Config.GlobalConfiguration GlobalConfiguration;
-        private List<string> SupportedImageFormats = new List<string>() { ".bmp", ".jpeg", ".jpg", ".tiff", ".tif", ".png", ".gif", ".emf", ".wmf", ".dwg", ".dicom", ".djvu" };
-        private List<string> SupportedDiagrammFormats = new List<string>() { ".vsd", ".vdx", ".vss", ".vsx", ".vst", ".vtx", ".vsdx", ".vdw", ".vstx", ".vssx" };
-        private static AnnotationImageHandler AnnotationImageHandler;
-        private DirectoryUtils DirectoryUtils;
+        private static Common.Config.GlobalConfiguration globalConfiguration;
+        private static AnnotationImageHandler annotationImageHandler;
+        private readonly List<string> supportedImageFormats = new List<string>() { ".bmp", ".jpeg", ".jpg", ".tiff", ".tif", ".png", ".gif", ".emf", ".wmf", ".dwg", ".dicom", ".djvu" };
+        private readonly List<string> supportedDiagrammFormats = new List<string>() { ".vsd", ".vdx", ".vss", ".vsx", ".vst", ".vtx", ".vsdx", ".vdw", ".vstx", ".vssx" };
```
Comment (egorovpavel) [inline src/Products/Annotation/Controllers/AnnotationApiController.cs:36]: Codacy Issue found: [Remove these redundant parentheses.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5681088)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 11
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-annotation/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5681087&bid=18596656)

File: src/Products/Annotation/Controllers/AnnotationApiController.cs [hunk 2]
```diff
@@ -12,168 +21,180 @@
...
             {
                 // set storage path
-                StoragePath = DirectoryUtils.FilesDirectory.GetPath()
-            };           
+                StoragePath = directoryUtils.FilesDirectory.GetPath(),
+            };
+
             // initialize Annotation instance for the Image mode
-            AnnotationImageHandler = new AnnotationImageHandler(config);
+            annotationImageHandler = new AnnotationImageHandler(config);
```
Comment (bobkovalex) [inline src/Products/Annotation/Controllers/AnnotationApiController.cs:57]: Codacy Issue found: [Remove this assignment of 'annotationImageHandler' or initialize it statically.](https://app.codacy.com/gh/groupdocs-annotation/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5681087)

Comment (egorovpavel) [inline src/Products/Annotation/Controllers/AnnotationApiController.cs:57]: Codacy Issue found: [Remove this assignment of 'annotationImageHandler' or initialize it statically.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5681088)

File: src/Products/Annotation/Controllers/AnnotationApiController.cs (hunk 1 above)
Comment (bobkovalex) [inline src/Products/Annotation/Controllers/AnnotationApiController.cs:36]: Codacy Issue found: [Remove these redundant parentheses.](https://app.codacy.com/gh/groupdocs-annotation/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5681087)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 10
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5681088&bid=18596657)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 12
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-annotation/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5681087&bid=18596656)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 10
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5681088&bid=18596657)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 12
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-annotation/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5681087&bid=18596656)

**Pull request merged**